<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>5</number>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="cover_label">
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="title_hlayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Title:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_title">
            <property name="text">
             <string>[title here]</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="author_hlayout">
          <item>
           <widget class="QLabel" name="author_title">
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Author:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_author">
            <property name="text">
             <string>[author here]</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="year_hlayout">
          <item>
           <widget class="QLabel" name="year_title">
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Year:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_year">
            <property name="text">
             <string>[year here]</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="publish_hlayout">
          <item>
           <widget class="QLabel" name="publish_title">
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Publishing Status:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_publish">
            <property name="text">
             <string>[publish here]</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="scan_hlayout">
          <item>
           <widget class="QLabel" name="scan_title">
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Scanlation Status:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_scan">
            <property name="text">
             <string>[scan here]</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="genre_hlayout">
          <item>
           <widget class="QLabel" name="genre_title">
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Genre:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_genre">
            <property name="text">
             <string>[genre here]</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="description_hlayout">
          <item>
           <widget class="QLabel" name="description_title">
            <property name="font">
             <font>
              <family>Arial Black</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Description:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="description_box">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Tahoma</family>
            <pointsize>9</pointsize>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stacked">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>10</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="manga_page">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QListWidget" name="manga_list"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="chapter_page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QTableView" name="chapter_table"/>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="btn_back_manga_list">
          <property name="text">
           <string>Back To Manga List</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuLibrary">
    <property name="title">
     <string>&amp;Library</string>
    </property>
    <widget class="QMenu" name="menuRead_Chapter">
     <property name="title">
      <string>Read Chapter</string>
     </property>
     <addaction name="action_single_page_viewer"/>
     <addaction name="action_two_page_viewer"/>
     <addaction name="action_web_viewer"/>
    </widget>
    <addaction name="action_remove_manga"/>
    <addaction name="separator"/>
    <addaction name="action_update_manga"/>
    <addaction name="action_update_library"/>
    <addaction name="separator"/>
    <addaction name="menuRead_Chapter"/>
    <addaction name="separator"/>
    <addaction name="action_set_selected_chapter_read"/>
    <addaction name="separator"/>
    <addaction name="action_download_chapters"/>
   </widget>
   <widget class="QMenu" name="menuWeb">
    <property name="title">
     <string>&amp;Web</string>
    </property>
    <addaction name="action_search_web"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLibrary"/>
   <addaction name="menuWeb"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_update_library"/>
   <addaction name="action_update_manga"/>
   <addaction name="separator"/>
   <addaction name="action_download_chapters"/>
   <addaction name="separator"/>
   <addaction name="action_single_page_viewer"/>
   <addaction name="action_two_page_viewer"/>
   <addaction name="action_web_viewer"/>
   <addaction name="separator"/>
   <addaction name="action_search_web"/>
  </widget>
  <action name="action_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_update_library">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/read_chapter/icons/update_library.png</normaloff>:/read_chapter/icons/update_library.png</iconset>
   </property>
   <property name="text">
    <string>Update Library</string>
   </property>
  </action>
  <action name="action_search_web">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/read_chapter/icons/search_web.png</normaloff>:/read_chapter/icons/search_web.png</iconset>
   </property>
   <property name="text">
    <string>Search The Web</string>
   </property>
  </action>
  <action name="action_single_page_viewer">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/read_chapter/icons/read_chapter.png</normaloff>:/read_chapter/icons/read_chapter.png</iconset>
   </property>
   <property name="text">
    <string>Single Page Viewer</string>
   </property>
  </action>
  <action name="action_two_page_viewer">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/read_chapter/icons/read_two_page.png</normaloff>:/read_chapter/icons/read_two_page.png</iconset>
   </property>
   <property name="text">
    <string>Two Page Viewer</string>
   </property>
  </action>
  <action name="action_web_viewer">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/read_chapter/icons/web_view.png</normaloff>:/read_chapter/icons/web_view.png</iconset>
   </property>
   <property name="text">
    <string>Web Page Viewer</string>
   </property>
  </action>
  <action name="action_update_manga">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/read_chapter/icons/update.png</normaloff>:/read_chapter/icons/update.png</iconset>
   </property>
   <property name="text">
    <string>Update Manga</string>
   </property>
  </action>
  <action name="action_add_from_url">
   <property name="text">
    <string>Add From Url</string>
   </property>
  </action>
  <action name="action_download_chapters">
   <property name="icon">
    <iconset resource="main_window.qrc">
     <normaloff>:/read_chapter/icons/download.ico</normaloff>:/read_chapter/icons/download.ico</iconset>
   </property>
   <property name="text">
    <string>Download Chapters</string>
   </property>
  </action>
  <action name="action_set_selected_chapter_read">
   <property name="text">
    <string>Set Selected Chapter As Read</string>
   </property>
  </action>
  <action name="action_remove_manga">
   <property name="text">
    <string>Remove Manga</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="main_window.qrc"/>
 </resources>
 <connections/>
</ui>
